specs/Standards/NonFungibleToken/payouts.md

```diff
@@ -0,0 +1,58 @@
+# Standard for a Multiple-Recipient-Payout mechanic on NFT Contracts
+
+A standard for supporting structures that allow for NFT's to inform financial
+contracts to payout multiple addresses on sales.
+
+Currently, NFT's on Near support the field `owner_id`, but lack flexibility for
```

Comment:
This is written from the perspective of the present, but is meant to be a reference document for the future. Should it be worded to tell people why they would want to use this NFT extension, rather than telling the repository maintainers why they should merge it?